apps/www/src/routes/examples/playground/(components)/model-selector.svelte

```diff
@@ -106,7 +106,7 @@
 											}}
 										>
 											<Command.Item
-												value={model.name}
+												value={model.id}
```

Comment:
This breaks the ability to search, try typing in `text` into the searchbar and you'll see.

```suggestion
												value={model.name}
```

What we should do instead to enable models to be selected is to update the `onSelect` callback to:


```ts
onSelect(() => {
	value = model.id
	closeAndFocusTrigger(ids.trigger)
})
```

Reply:
Done, thanks for keeping an eye out. Will be more careful in the future.